userChrome-default.css

```diff
@@ -35,11 +35,11 @@
 
 /* Styles to account for when tab scrollbar is visible, and margin-inline-start kicks in */
 .tabbrowser-tab[pinned="true"] {
-  width: 40px !important; /* Actual photon pinned tab is 41px, but 40px removes pixelated artifact from pinned tab favicon */
+  width: 34px !important;
 }
 
 .tabbrowser-tab[pinned="true"][style^="margin-inline-start"] {
-  padding-left: 12px !important;
+  padding-left: 8px !important;
```

Comment:
These changes look to be separate from supporting theme-detection. I'll give them a try and see if these numbers work.

Reply:
yeah, I've added it to center the icons on pinned tabs:
![Screen Shot 2019-11-13 at 16 35 02](https://user-images.githubusercontent.com/466367/68797545-8f918980-0633-11ea-91a2-7f81b1c6c6d4.png)